api-specifications.md

```diff
@@ -96,6 +104,164 @@ Create a course (anyone logged in). Used for outside Exchange.
 * Login required
 * Course already exists
 
+### /api/instructor: Course instructor management
+
+#### POST /api/instructor/&lt;course_id&gt;/&lt;instructor_id&gt;
+*Add an instructor to the course. (instructors only)*
+
+##### Request (HTTP POST data)
+```
+first_name=/*instructor name*/&
+last_name=/*instructor last name*/&
+email=/*instructor email*/
+```
+
+##### Response
+```javascript
+{
+    "success": true
+}
+```
+
+##### Error messages
+* Login required
+* Permission denied
+* Course not found
+
+#### GET /api/instructor/&lt;course_id&gt;/&lt;instructor_id&gt;
+*Gets information about a course instructor. (instructors+students)*
+
+##### Response
+```javascript
+{
+    "success": true,
+    "username": /* instructor ID */,
+    "first_name": /* instructor name*/,
+    "last_name": /* instructor last name*/,
+    "email": /* instructor email*/
+}
+```
+
+##### Error messages
+* Login required
+* Permission denied
+* Course not found
+* Instructor not found
+* Removing self
```

Comment:
"Removing self" should be in next section (around line 160, here is 150). 
Maybe we should use what Lawrence suggests (use "Removing last instructor from course")